ods_ci/tests/Resources/Common.robot

```diff
@@ -314,6 +314,17 @@ Skip If RHODS Is Managed
        Skip If    condition=${is_self_managed}==False    msg=This test is skipped for Managed RHODS
     END
 
+Skip If RHODS Is Self-Managed And New Observability Stack Is Disabled
+    [Documentation]    Skips test if RHODS is installed as Self-managed and ${ENABLE_NEW_OBSERVABILITY_STACK} == false
+    [Arguments]    ${msg}=${EMPTY}
+    ${is_self_managed}=    Is RHODS Self-Managed
+    ${enable_new_observability_stack} =    Is New Observability Stack Enabled   # Replace with this also other occurrences
```

Comment:
What does the comment mean? Is this a TODO that needs to be done once the rest of the functionality is implemented? I see this being used everywhere that I can tell